### ISSUE TITLE
`pop new pallet`: in folder pallets

### ISSUE THREAD

**Issue by AlexD10S** [r0gue-io/pop-cli#58]
Currently when adding `pop new pallet` it just add it in the directory you are located by default.

A cool feature would be to detect if you are running this inside a node folder, and  ask the user if wants to create it inside a pallet folder.
 If yes: 

1. create a pallet folder
2.  create the new pallet in this folder
3.  add the workspace pallet code in the root directory `Cargo.toml`:

```rs
[workspace]
members = [
	"node",
	"runtime",
         "pallets/*",
]
```

If this line is not added into `Cargo.toml` it will failled to build the pallet:
```
error: current package believes it's in a workspace when it's not:
current:   /myparachain/pallet-template/Cargo.toml
workspace: /myparachain/Cargo.toml

this may be fixable by adding `pallet-template` to the `workspace.members` array of the manifest located at: /mytest/Cargo.toml
Alternatively, to keep it out of the workspace, add the package to the `workspace.exclude` array, or add an empty `[workspace]` table to the package's manifest.
```
My concern here is if the user only want to create a pallet template, not adding it into a node folder. Would this question make sense to them?

**Comment by weezy20** [comment]
maybe closed by #62?

**Comment by AlexD10S** [comment]
Close with https://github.com/r0gue-io/pop-cli/pull/277
